src/SizeBench.LocalBuild/LocalBuildPathLocator.cs

```diff
@@ -40,4 +40,30 @@ public bool TryInferBinaryPathFromPDBPath(string pdbPath, out string binaryPath)
         binaryPath = String.Empty;
         return false;
     }
-}
\ No newline at end of file
+
+    public bool TryInferPDBPathFromBinaryPath(string binaryPath, out string pdbPath)
+    {
+        if (binaryPath is null)
+        {
+            pdbPath = String.Empty;
+            return false;
+        }
+
+        var possiblePdbPath = Path.ChangeExtension(binaryPath, "pdb");
+        if (File.Exists(possiblePdbPath))
+        {
+            pdbPath = possiblePdbPath;
+            return true;
+        }
+
+        possiblePdbPath += ".pdb";
```

Comment:
>possiblePdbPath += ".pdb"; [](http://example.com/codeflow?start=8&length=26)

I don't think this works, since `possiblePdbPath` above is now `foo.pdb` due to `ChangeExtension`

So probably this should be:

`possiblePdbPath = binaryPath + ".pdb"`

?